packages/mrc-ide-covidsim/test/integration/params-integration-test.ts

```diff
@@ -0,0 +1,152 @@
+        },
+        {
+          startDate: '2020-03-31',
+          reductionPopulationContact: 78,
+          caseIsolation: input.Intensity.Aggressive,
+          schoolClosure: input.Intensity.Aggressive,
+          socialDistancing: input.Intensity.Aggressive,
+          voluntaryHomeQuarantine: input.Intensity.Aggressive,
+        },
+      ],
+    },
+  }
+
+  const logDir = temp.mkdirSync()
+  const inputDir = temp.mkdirSync()
+  const outputDir = temp.mkdirSync()
+  const binDir = temp.mkdirSync()
+
+  const model = new ImperialModel(
+    1,
+    binDir,
+    logDir,
+    MODEL_DATA_DIR,
+    inputDir,
+    outputDir
+  )
+
+  const runnerModelInput = model.inputs(modelInput)
+
+  test('validate US-CA admin file', () => {
+    const adminParamsText = fs.readFileSync(
+      runnerModelInput.adminFilePath,
+      'utf8'
+    )
+    const adminParams = parse(adminParamsText)
+
+    assert.include(adminParams, {
+      ['Include holidays']: 0,
+      ['Fix population size at specified value']: 0,
+      ['Number of countries to include']: 0,
+      ['Number of level 1 administrative units to include']: 1,
+      ['List of level 1 administrative units to include']: 'California',
+    })
+  })
+
+  test('validate US-CA pre-param file', () => {
+    const preParamsText = fs.readFileSync(
+      runnerModelInput.preParametersFilePath,
+      'utf8'
+    )
+    const preParams = parse(preParamsText)
+
+    assert.doesNotHaveAnyKeys(preParams, [
+      'Number of detected cases needed before outbreak alert triggered',
+    ])
+
+    // Validate static settings
+    assert.include(preParams, {
+      ['Number of days to accummulate cases/deaths before alert']: 1000,
+      ['Trigger alert on deaths']: 1,
+      ['Alert trigger starts after interventions']: 1,
+      ['Treatment trigger incidence per cell']: 0,
+      ['Places close only once']: 0,
+      ['Social distancing only once']: 0,
+    })
+
+    // Validate dynamic settings
+    assert.include(preParams, {
+      ['Day of year trigger is reached']: 130,
+      ['Number of deaths accummulated before alert']: 2678,
+      ['Day of year interventions start']: 79,
+    })
+  })
+
+  test('validate US-CA params file', () => {
+    const paramsText = fs.readFileSync(
+      runnerModelInput.parametersFilePath,
+      'utf8'
+    )
+    const params = parse(paramsText)
+
+    // Validate static settings
+    assert.deepInclude(params, {
+      ['Vary efficacies over time']: 1,
+      ['Place closure incidence threshold']: 0,
+      ['Place closure fractional incidence threshold']: 0,
+      ['Household quarantine trigger incidence per cell']: 0,
+      ['Case isolation trigger incidence per cell']: 0,
+      ['Trigger incidence per cell for place closure']: 0,
+      ['Trigger incidence per cell for place closure over time']: [0, 0],
+      ['Trigger incidence per cell for social distancing']: 0,
+      ['Trigger incidence per cell for social distancing over time']: [0, 0],
```

Comment:
Would be nice to include ensuring that the `Relative place contact rates over time given enhanced social distancing by place type` parameter is set properly. It is a multi-dimensional array and should all be set to 0.